pdal/util/FileUtils.cpp

```diff
@@ -234,7 +234,11 @@ std::vector<std::string> directoryList(const std::string& dir)
         fs::directory_iterator end;
         while (it != end)
         {
+#ifndef _MSC_VER
```

Comment:
This should probably be PDAL_WIN32_STL as well.

Reply:
yes